api/Performance.json

```diff
@@ -50,48 +48,440 @@
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "mark": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/mark",
+          "support": {
+            "webview_android": {
+              "version_added": "46"
+            },
+            "chrome": {
+              "version_added": "43"
+            },
+            "chrome_android": {
+              "version_added": "46"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "41"
+            },
+            "firefox_android": {
+              "version_added": "42"
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "33"
+            },
+            "opera_android": {
+              "version_added": "33"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "measure": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/measure",
+          "support": {
+            "webview_android": {
+              "version_added": "46"
+            },
+            "chrome": {
+              "version_added": "43"
+            },
+            "chrome_android": {
+              "version_added": "46"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "41"
+            },
+            "firefox_android": {
+              "version_added": "42"
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "33"
+            },
+            "opera_android": {
+              "version_added": "33"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": "10"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "memory": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Performance/memory",
+          "support": {},
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```

Comment:
`Performance.memory` is a non-standard extension according to the [MDN article](https://developer.mozilla.org/en-US/docs/Web/API/Performance#Properties).